api/integrations/fifabot/integrations/slack.js

```diff
@@ -0,0 +1,15 @@
+const Botkit = require('botkit');
```

Comment:
I think we need a better name for this file. This is the setup file needed for all slack integrations configured, so we could rename as such.

Reply:
botkit is a package that interfaces with slack API for us. what can be done is renamed just the variable, not the file

Reply:
I wasn't talking about the line itself, but the file name.

I meant that this is not an integration by itself, so it should be renamed. We can leave that as it is right now since I can't think of a better name ("slack_setup", "base_slack", or something like that, idk).